bin/builder-core.js

```diff
@@ -14,6 +14,7 @@ var log = require("../lib/log");
  * @param {Object}    [opts]      Options object
  * @param {Object}    [opts.env]  Environment object to mutate (Default `process.env`)
  * @param {Array}     [opts.argv] Arguments array (Default: `process.argv`)
+ * @param {Array}     [opts.msgs] Array of log messages.
```

Comment:
Am I correct in understanding that these are not log messages but message templates?